pkg/controller/v1alpha2/applicationconfiguration/render.go

```diff
@@ -659,3 +674,32 @@ func getTraitName(ac *v1alpha2.ApplicationConfiguration, componentName string,
 
 	return traitName
 }
+
+// getCurrentWorkload tries to retrieve the currently running workload
+func (r *components) getCurrentWorkload(ctx context.Context, ac *v1alpha2.ApplicationConfiguration, c *v1alpha2.Component, w *unstructured.Unstructured) *unstructured.Unstructured {
```

Comment:
I don't think errors except `*net.OpError`  all  mean 'the workload not exist', what do you think? Why don't just return the erorr?

Reply:
Yes, it just means we could not find the workload due to network issues